src/core/infer_response.h

```diff
@@ -137,6 +137,9 @@ class InferenceResponse {
     // The shape of the output tensor.
     const std::vector<int64_t>& Shape() const { return shape_; }
 
+    // Reshape the output tensor if specified in output config.
+    void Reshape(const ModelOutput* output_config);
```

Comment:
This can be private

Reply:
`Rehape()` is member function of `InferenceResponse::Output` and called in `InferenceResponse`